### ISSUE TITLE
How to use ONNX to implement tinyYolo on hololens ?

### ISSUE THREAD

**Issue by valavakilian** [onnx/models#134]
Hi. I'm trying to run both segmentation and object detection on a Hololens device.
I am able to access the research mode for Hololens but I cannot do on device inference because most of my trained models are in pytorch. I wanted to see if ONNX can help me with this task.
I understand that the mask-RCNN and Yolov2 are not developed for ONNX so far. But I can't seem to understand how to run the tinyYolo model either ? 
Thank you

**Comment by prasanthpul** [comment]
You can get the pre-trained TinyYOLO model from the ONNX model zoo: https://github.com/onnx/models/tree/master/tiny_yolov2
You can run ONNX models on HoloLens using WinML: https://docs.microsoft.com/en-us/windows/ai/

Support for more object detection models is in the works and PyTorch export is also being improved.
